charts/cosi/values.yaml

```diff
@@ -132,19 +58,6 @@ resources:
   #   cpu: 100m
   #   memory: 128Mi
 
-# autoscaling rules for COSI driver deployment.
-autoscaling:
-  # enabled specifies whether autoscaling is enabled.
-  enabled: false
-  # minReplicas specifies the minimum number of replicas.
-  minReplicas: 1
-  # maxReplicas specifies the maximum number of replicas.
-  maxReplicas: 100
-  # targetCPUUtilizationPercentage specifies the target CPU utilization percentage.
-  targetCPUUtilizationPercentage: 80
-  # targetMemoryUtilizationPercentage specifies the target memory utilization percentage.
-  # targetMemoryUtilizationPercentage: 80
-
 # nodeSelector specifies the node selector for the COSI driver pod.
 nodeSelector: {}
 # tolerations specifies the list of tolerations for the COSI driver pod.
```

Comment:
Are we removing affinity to align with other drivers?

Reply:
Updated